bom/runtime/pom.xml

```diff
@@ -173,9 +173,9 @@
         <sundr.version>0.19.1</sundr.version> <!-- this is to avoid annoying pop-up in eclipse about failure to init Velocity logging -->
         <flapdoodle.mongo.version>2.2.0</flapdoodle.mongo.version>
         <wiremock.version>2.24.1</wiremock.version>
-        <quarkus-spring-api.version>5.2.Final</quarkus-spring-api.version>
+        <quarkus-spring-api.version>5.3.Alpha1-SNAPSHOT</quarkus-spring-api.version>
```

Comment:
How does this one get updated to 5.3? Spring Core is still 5.2 AFAICS.

Reply:
Yes, see my comment

Reply:
Fixed